snapper-samples/build.gradle

```diff
@@ -22,5 +22,5 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:appcompat-v7:23.0.1'
```

Comment:
Doesn't that make this a transitive dependency that would have to be declared somehow?